### ISSUE TITLE
Material for the model is all black

### ISSUE THREAD

**Issue by ohthatsjhall** [googlevr/poly-sample-ios#1]
Tried adding the model to my SceneKit view, however the material is completely black, which I assume is because the material has not been assigned to the model. I tried running on the SceneKit sample project and got the same result.

I've tried with a few different assets and I will get a light gray or solid black mesh. 

 Any suggestions?

**Comment by nirav** [comment]
Do the assets that you're downloading have an MTL file?

**Comment by ohthatsjhall** [comment]
Yes I am using the Poly API. Here are a few examples of urls containing the .mtl that I've tried

https://poly.googleapis.com/downloads/0t2ZYRBsqX-/d6v3eGvAZ8h/Lighthouse%20singly.mtl
https://poly.googleapis.com/downloads/eke7qcu_FR2/50NqG3OK3rn/Hamburger.mtl
https://poly.googleapis.com/downloads/5vbJ5vildOq/5LkfC03mUMN/materials.mtl

the last url is the one provided in the sample project.

**Comment by nirav** [comment]
Looking into this! Apologies for the delay.
